src/main/java/frc/robot/commands/swervedrive/drivebase/TeleopDrive.java

```diff
@@ -67,6 +86,12 @@ public void execute()
     
 
     // Drive using raw values.
+    if(isFaceSpeaker)
+    {
+      PID.setSetpoint(UtilMath.SpeakerTheta(swerve.getPose()));
```

Comment:
I think you changed the method name to FrontSpeakerTheta/BackSpeakerTheta. Getting a compile error.